modules/search/instant-search/components/search-box.scss

```diff
@@ -0,0 +1,12 @@
+label.jp-instant-search__box input:focus {
+}
+
+/* apply to all the inputs to try and pick up any theme styling */
+.jp-instant-search__box input {
+	-moz-border-radius: 2px;
```

Comment:
Do we have a `border-radius` mixin we can use here instead of using vendor prefixes?

Reply:
Build pipeline should add these automatically, so they don't need to be here.

Reply:
@simison so I can just delete that rule completely then?

Reply:
Yep safe to remove. They're added in build with Autoprefixer if they're needed based on our browserlist-versions that we support.

https://github.com/postcss/autoprefixer